docroot/sites/facilities.uiowa.edu/modules/facilities_core/src/Form/NodeAlertDispatchForm.php

```diff
@@ -0,0 +1,160 @@
+      $container->get('renderer'),
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'node_alert_dispatch_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, NodeInterface $node = NULL) {
+    $config = $this->config('facilities_core.settings');
+    $this->node = $node;
+
+    if (is_null($this->dispatch->getKey())) {
+      $form['no_api_key'] = [
+        '#markup' => $this->t('A Dispatch API key has not been entered. Please add your API key.'),
+      ];
+
+      return $form;
+    }
+
+    $communication_id = $config->get('alert_dispatch_communication_id');
+
+    if (!$communication_id) {
+      $form['no_communication_id'] = [
+        '#markup' => $this->t('A Dispatch communication ID has not been entered. Please select a communication ID in settings.'),
+      ];
+
+      return $form;
+    }
+
+    $form['title'] = [
+      '#markup' => $this->t('<h3>Schedule Email Communication</h3>'),
+    ];
+
+    $form['start'] = [
+      '#type' => 'datetime',
+      '#title' => $this->t('Send date and time'),
+      '#required' => TRUE,
+      '#date_increment' => 60,
+    ];
+
+    $form['placeholders'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Placeholders'),
+      '#description' => $this->t('These placeholders will be used to fill in the message that is sent.'),
+      '#open' => FALSE,
+    ];
+
+    $placeholders = $this->getPlaceholders();
+
+    foreach ($placeholders as $placeholder => $preview) {
+
+      $form['placeholders'][$placeholder]['label'] = [
+        '#type' => 'label',
+        '#title' => $placeholder,
+      ];
+
+      if (!empty($preview)) {
+        $form['placeholders'][$placeholder]['value'] = [
+          '#markup' => $preview,
+        ];
+      }
+    }
+
+    $form['submit'] = [
+      '#type' => 'submit',
+      '#value' => $this->t('Send Dispatch request'),
+    ];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    $config = $this->config('facilities_core.settings');
+    $schedule_start = strtotime($form_state->getValue('start'));
+
+    $communication_id = $config->get('alert_dispatch_communication_id');
+
+    $placeholders = $this->getPlaceholders();
+
+    $this->dispatch->postCommunicationSchedule($communication_id, $schedule_start, $placeholders);
+
+    $this->messenger()->addMessage($this->t('Message request has been sent.'));
+  }
+
+  /**
+   * Prepare placeholders for display.
+   *
+   * @return array
+   *   The rendered placeholder values keyed by placeholder name.
+   */
+  protected function getPlaceholders() {
+    $placeholders = [];
+
+    foreach (_sitenow_dispatch_get_placeholders('alert') as $field_name => $placeholder) {
+      switch ($field_name) {
+        case 'alert_subject':
+          $placeholders[$placeholder] = $this->node->getTitle() . ' - OSC TEST';
```

Comment:
Should this be updated to remove `- OSC TEST`

Should there be some kind of other "test" setup to toggle that adds this?

Reply:
```suggestion
          $placeholders[$placeholder] = $this->node->getTitle();
```